src/main/java/appeng/parts/reporting/AbstractPartTerminal.java

```diff
@@ -115,19 +115,12 @@ public boolean onPartActivate( final EntityPlayer player, final EnumHand hand, f
 	{
 		if( !super.onPartActivate( player, hand, pos ) )
 		{
-			if( !player.isSneaking() )
+			if( Platform.isServer() )
 			{
-				if( Platform.isClient() )
-				{
-					return true;
-				}
-
 				Platform.openGUI( player, this.getHost().getTile(), this.getSide(), this.getGui( player ) );
-
-				return true;
 			}
 		}
-		return false;
+		return true;
 	}
 
 	public GuiBridge getGui( final EntityPlayer player )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "player". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1172)